draft-ietf-quic-transport.md

```diff
@@ -1612,8 +1611,8 @@ sender or during delivery to the application at the receiver.
 When new data is to be sent on a stream, a sender MUST set the encapsulating
 STREAM frame's offset field to the stream offset of the first byte of this new
 data.  The first byte of data that is sent on a stream has the stream offset 0.
-A receiver MUST ensure that received stream data is delivered to the application
-as an ordered byte-stream.  Data received out of order MUST be buffered for
+A receiver MUST ensure that received stream data creates an ordered
+byte-stream.  Data received out of order may need to be buffered for
```

Comment:
I believe the purpose of the MUST was to ensure that if a packet with a stream frame is acked, it's not discarded.  The new wording is somewhat ambiguous to me, though I know what you're aiming for.

Also, can we remove "as long as it is not in violation of the receiver's flow control limits."  That causes a connection close error, and mentioning flow control so briefly isn't that helpful.